resources.Groups.department.ReferenceExample.ps1

```diff
@@ -4,17 +4,124 @@ $success = $false # Set to false at start, at the end, only when no error occurs
 
 $auditLogs = [Collections.Generic.List[PSCustomObject]]::new()
 
+#$AFK = $rRef.SourceData.ADgroepAFK
+#$Class = $p.PrimaryContract.Custom.educationalGroup
+#$ou = $p.primaryContract.custom.OU
+
+#Write-Verbose -Verbose $resourceContext
+
 # Troubleshooting
 # $dryRun = $false
```

Comment:
hadden we niet besloten dat we dit weg zouden laten. Maakt me verder persoonlijk niet uit.